src/ui/components/shared/Login/Login.tsx

```diff
@@ -104,20 +97,11 @@ function SocialLogin({
           </a>
         </div>
       </div>
-      <PrimaryLgButton color="blue" onClick={onLogin} className="w-full justify-center">
```

Comment:
@jaril , @jonbell-lot23 - this is changing #6493 a bit but hopefully doesn't undo what you were setting up there. Let me know and I can help refactor

Reply:
Nah I think this is good! The downsides of not being DRY 🤣